<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnAddBattery">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>280</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Batteryn</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelVoltage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Voltage(V)</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spinVoltage">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>390</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximum">
     <double>100.000000000000000</double>
    </property>
    <property name="value">
     <double>3.700000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="labelCapacity">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>330</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Capacity (mAh)
</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spinCapacity">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>330</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximum">
     <double>10000.000000000000000</double>
    </property>
    <property name="value">
     <double>2000.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="labelInitialCharge">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Initial Charge (mAh)
</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spinInitialCharge">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>340</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximum">
     <double>10000.000000000000000</double>
    </property>
    <property name="value">
     <double>1500.000000000000000</double>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>440</y>
      <width>311</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Connection Type </string>
    </property>
    <widget class="QRadioButton" name="radioSeries">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Series</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioParallel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Paralel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnRecharge">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Recharge (1 hour)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnUse">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Use (1 hour)
</string>
     </property>
    </widget>
   </widget>
   <widget class="BatteryCanvas" name="drawArea" native="true">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>6</y>
      <width>788</width>
      <height>89</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BatteryCanvas</class>
   <extends>QWidget</extends>
   <header>batterycanvas.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
